### PR TITLE
Design tweaks for DM Room Tiles

### PR DESCRIPTION
**Pull request by t3chguy**
+ Tweaks UserOnlineDot and pulls it out of Labs
+ Nukes the DM Indicator on the RoomTile and on breadcrumbs

Further tweaking may be done in the planned RoomList redesign.
There is no design for showing it in the collapsed state, so it simply won't show there.
It occupies the same space as the `...` context menu trigger so will disappear during hover.

uod
image

Fixes https://github.com/vector-im/riot-web/issues/11799

### CODE REVIEW

File: src/components/views/rooms/RoomBreadcrumbs.js [hunk 1]
```diff
@@ -363,17 +363,6 @@ export default class RoomBreadcrumbs extends React.Component {
...
-            if (this._isDmRoom(r.room) && !SettingsStore.isFeatureEnabled("feature_cross_signing")) {
-                dmIndicator = <img
-                    src={require("../../../../res/img/icon_person.svg")}
-                    className="mx_RoomBreadcrumbs_dmIndicator"
-                    width="13"
-                    height="15"
-                    alt={_t("Direct Chat")}
-                />;
-            }
-
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply src/components/views/rooms/RoomBreadcrumbs.js:376]: image

@nadonomy should this remain for breadcrumbs or vanish everywhere?

File: src/settings/Settings.js [hunk 2]
```diff
@@ -131,12 +131,6 @@ export const SETTINGS = {
         supportedLevels: LEVELS_FEATURE,
         default: false,
     },
-    "feature_presence_in_room_list": {
-        isFeature: true,
-        displayName: _td("Show a presence dot next to DMs in the room list"),
-        supportedLevels: LEVELS_FEATURE,
-        default: false,
-    },
```
[the remark this answers is not in the document]
Comment (t3chguy) [reply src/settings/Settings.js:139]: image

[the remark this answers is not in the document]
Comment (turt2live) [reply src/settings/Settings.js:139]: still need an explicit checkmark here, as much fun as discussing it internally is.

File: src/components/views/rooms/RoomBreadcrumbs.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (nadonomy) [reply src/components/views/rooms/RoomBreadcrumbs.js:376]: For posterity— totally fine to not display the DM badge in breadcrumbs. The breadcrumbs are just simple shortcuts for convenience, and the DM icons are part of the old icon language from years ago.

**Pull request merged**